package/MDAnalysis/visualization/streamlines.py

```diff
@@ -85,7 +86,8 @@ def produce_grid(tuple_of_limits, grid_spacing):
     return grid
 
 
-def split_grid(grid, num_cores):
+# typing: numpy
+def split_grid(grid: np.ndarray, num_cores: int) -> List[List[List[int]], List[List[int]], int, int]:
```

Comment:
Are the 2 first return values really lists of lists? The docstring talks about arrays of arrays, and the numpy doc let me think they may be lists of arrays (<https://numpy.org/doc/stable/reference/generated/numpy.split.html>). Anyway, the docstring and/or the annotation need fixing.